check_http_json.rb

```diff
@@ -18,23 +18,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-
 # Requires.
-require 'rubygems'
```

Comment:
Really? I didint seen a required binding to the rubygems lib. Do you know what breaks in 1.8.7?